### PR TITLE
docs - add pageinspect module landing page

### PR DESCRIPTION
**Pull request by lisakowen**
unify the organization and content of the docs for additional supplied modules. for pageinspect:
- create a landing page for the module docs
- identify that the module is equivalent to postgresql, no greenplum considerations
- xref to module install/registration page
- xref to postgres pageinspect module docs

### CODE REVIEW

**Comment by Eulerizeit** [comment]
I don't think we are planning to ship this. Also, I assume this works with HEAP tables only right now.

**Comment by lisakowen** [comment]
i installed the 6.0 beta.1 RPM.  the library and extension files for the pageinspect module are included, and i can "CREATE EXTENSION pageinspect;" in a database.

File: gpdb-doc/dita/ref_guide/modules/intro.xml [hunk 1]
```diff
@@ -33,9 +33,9 @@
        <li><codeph><xref href="orafce_ref.xml" scope="peer" format="dita"
              >orafce</xref></codeph> - Provides Greenplum Database-specific Oracle
           SQL compatibility functions.</li>
-       <li><codeph><xref href="https://www.postgresql.org/docs/9.4/pageinspect.html"
-             scope="external" format="html">pageinspect</xref></codeph> - Functions for low level
-         inspection of the contents of database pages; available to superusers only.</li>
+       <li><codeph><xref href="pageinspect.xml" scope="peer" format="dita"
```
Comment (litzell) [inline gpdb-doc/dita/ref_guide/modules/intro.xml]: why is scope peer and not local? I'm not sure it matters, but I thought peer was for when the linked source file isn't present at build time.

Comment (lisakowen) [reply gpdb-doc/dita/ref_guide/modules/intro.xml]: thanks for reviewing, chuck!  those scopes were remnants of a copy-paste.  i have removed a few unneeded references from this file.

**Comment by lisakowen** [comment]
@danielgustafsson - would you happen to know if there are any greenplum- or mpp- specific considerations for this module?  a previous comment mentioned that this might work only with HEAP tables.

thanks.

File: gpdb-doc/dita/ref_guide/modules/pageinspect.xml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+<dita>
+  <topic id="topic_fstrm">
+    <title>pageinspect</title>
+    <body>
+      <p>The <codeph>pageinspect</codeph> module provides functions for low level
+         inspection of the contents of database pages. <codeph>pageinspect</codeph>
+         is available only to Greenplum Database superusers.</p>
+      <p>The Greenplum Database <codeph>pageinspect</codeph> module is equivalent
+        to the PostgreSQL <codeph>pageinspect</codeph> module. There are no
+        Greenplum Database or MPP-specific considerations for the module.</p>
```
Comment (danielgustafsson) [inline gpdb-doc/dita/ref_guide/modules/pageinspect.xml]: This module does indeed not handle Append-Optimized or External relations, which probably should be mentioned.

**Pull request merged**